tests/integration/aps/test_aps_api_client.py

```diff
@@ -0,0 +1,44 @@
+)
+
+
+def test_get_articles_metadata(aps_api_client_fixture):
+    parameters = APSParams(
+        from_date="2021-04-10", until_date="2022-04-10", per_page=1
+    ).get_params()
+    metadata_value = aps_api_client_fixture.get_articles_metadata(parameters=parameters)
+    assert type(metadata_value) == type(expected_metadata_value)
+    assert metadata_value == expected_metadata_value
+
+
+def test_get_pdf_file(aps_api_client_fixture):
+    doi = "10.1103/PhysRevLett.126.153601"
+    parameters = APSParams(from_date="2021-04-10", until_date="2022-04-10").get_params()
+    pdf = aps_api_client_fixture.get_pdf_file(parameters=parameters, doi=doi)
```

Comment:
this test runs directly into the APS server, we shouldn't do that because:
* if the APS goes down our tests are relying on that and will fail as well
* We will create unnecessary traffic to their server

Hence, we should mock the response from APS. To do that you should use https://vcrpy.readthedocs.io/en/latest/, let me know if you would like to discuss it IRL

This applies to all tests that are getting data from APS servers

Reply:
so integration test has to mock response as well?

Reply:
yes, any external service should be mocked. Please use `vcrpy` to do that

Reply:
This still don't have a cassette